react/admin/OrganizationsList.tsx

```diff
@@ -50,199 +28,17 @@ export const labelTypeByStatusMap: Record<string, string> = {
   'on-hold': 'warning',
 }
 
-const initialState = {
-  status: ['active', 'on-hold', 'inactive'],
-  search: '',
-  page: 1,
-  pageSize: 25,
-  sortOrder: 'ASC',
-  sortedBy: 'name',
-}
-
 const OrganizationsList: FunctionComponent = () => {
   const { formatMessage } = useIntl()
-  const {
-    navigate,
-    culture: { country },
-  } = useRuntime()
-
-  const showToast = useToast()
+  const { navigate } = useRuntime()
 
   const [filterState, setFilterState] = useState({
     filterStatements: [] as FilterStatement[],
   })
 
-  const [loadingState, setLoadingState] = useState(false)
-  const [variableState, setVariables] = useState(initialState)
-  const [newOrganizationModalState, setNewOrganizationModalState] = useState(
-    false
-  )
-
-  const [newOrganizationName, setNewOrganizationName] = useState('')
-  const [newCostCenterName, setNewCostCenterName] = useState('')
-  const [newCostCenterPhoneNumber, setNewCostCenterPhoneNumber] = useState('')
-  const [
-    newCostCenterBusinessDocument,
-    setNewCostCenterBusinessDocument,
-  ] = useState('')
-
-  const [
-    newCostCenterStateRegistration,
-    setNewCostCenterStateRegistration,
-  ] = useState('')
-
-  const [newCostCenterAddressState, setNewCostCenterAddressState] = useState(
-    addValidation(getEmptyAddress(country))
-  )
-
-  const { data, loading, refetch } = useQuery(GET_ORGANIZATIONS, {
-    variables: initialState,
-    ssr: false,
-  })
```

Comment:
Except for this block of code, it was moved to `react/organizations/hooks.ts` instead, so I could reuse the fetching/refetching logic in different components.